CONTRIBUTING.md

```diff
@@ -4,13 +4,45 @@
 
 Please note we have a code of conduct, please follow it in all your interactions with the project.
 
-## Pull Request Process
+# Pull Request Process
+
+## Steps for the PR author
 
 1. Ensure that tests pass and code is lint free. You can run `yarn test` and `yarn lint` locally to check.
-2. Update the README.md if any changes invalidate its current content.
+2. Update the README.md if any changes invalidate or extend its current content.
 3. Include tests for any new functionality.
 4. Reference relevant issues in your PR comment.
```

Comment:
We should be explicit around proving context in the PR comment

Reply:
What I mean is actually respecting the PR template (filling it out)

Reply:
I changed the last point in the list above to:
> Ensure each section of the [Pull Request Template](./PULL_REQUEST_TEMPLATE.md) is filled out.